src/sentry/templates/sentry/js-sdk-loader.js.tmpl

```diff
@@ -111,11 +111,25 @@
         var integrationNames = integrations.map(function (integration) { return integration.name; });
         // Add necessary integrations based on config
         if (config.tracesSampleRate && integrationNames.indexOf('BrowserTracing') === -1) {
-            integrations.push(new SDK.BrowserTracing());
+            if (SDK.browserTracingIntegration) {
+                // (Post-)v8 version of the BrowserTracing integration
+                integrations.push(SDK.browserTracingIntegration());
+            }
+            else {
+                // Pre v8 version of the BrowserTracing integration
+                integrations.push(new SDK.BrowserTracing());
+            }
```

Comment:
As discussed let's flip the switch here to avoid (unlikely but still) breaking anyone expecting that the integration is a class.

Reply:
You are absolutely right. I adjusted the logic to use the old integration if it exists and the new one if the old one doesn't exist.